### DISCUSSION TITLE
"Downtime" between Monthly Postings

### DISCUSSION THREAD

**Discussion by Nio-1** [CMSgov/price-transparency-guide#312] [Q&A] [answered]
Will "downtime" between posting our monthly files be tolerated by CMS? For example, our July 2022 file is live and available for consumption. When we are doing the internal work to post the August 2022 file, will reasonable downtime to move the files behind the URLs, upload/download times, etc. be tolerated to allow for the new file to be posted? It is assumed by us that transitioning from one file to the other would require time and we are estimating that this "downtime" might be a couple of minutes to under a hour. However, we have not seen any discussion that this "work" violates the availability of the files as they are updated monthly.

**Comment by tparency** [comment]
Many rely on cloud vendors and while they have good uptime, we're beholden to their infrastructure and issues like last week's AWS outage would be an unforeseen issue. S3 has 99.9% and in reality that's 8 hours yearly. 

Last week had a region outage of more than 8hrs, there was another region outage yesterday. If this was after 7/1/22, would this outage cause a penalty? With large file sizes, what about degraded service? 

What guidance can CMS provide about how plans will be measured with these scenarios?

**Comment by shaselton-usds** [comment, marked as answer]
There are always going to be situations where downtimes are unscheduled due to outages and the sort -- those can be hard to plan for (though there are certainly many strategies out there that mitigate being completely down). 

Actual scheduled downtime should be and in many cases can be avoided - it presents a bad user experience. At this moment there isn't any threshold of downtime that is allowed, though this can be revisited if it started to become a problem.
